modules/ui/maproulette_details.js

```diff
@@ -0,0 +1,319 @@
+            }
+            if (allProps.has(propertyName)) {
+                const val = allProps.get(propertyName);
+                return val !== undefined && val !== null ? String(val) : '';
+            }
+            // Keep mustache tag visible if not replaced (Rapid behavior)
+            return match;
+        });
+    }
+
+    function render(selection) {
+        let details = selection
+            .selectAll('.error-details')
+            .data(_qaItem ? [_qaItem] : [], (d) => d.key);
+        details.exit().remove();
+
+        const detailsEnter = details
+            .enter()
+            .append('div')
+            .attr('class', 'error-details');
+
+        // 1) qa-header (icon + label)
+        const headerEnter = detailsEnter
+            .append('div')
+            .attr('class', 'qa-header');
+
+        const svgEnter = headerEnter
+            .append('div')
+            .attr('class', 'qa-header-icon')
+            .append('svg')
+            .attr('width', '20px')
+            .attr('height', '30px')
+            .attr('viewBox', '0 0 20 30')
+            .attr(
+                'class',
+                (d) =>
+                    `preset-icon-28 qaItem ${d.service} itemId-${d.id} itemType-${d.itemType}`,
+            );
+
+        // Header icon: circular head with tail, color #47725f, border black
+        svgEnter
+            .append('circle')
+            .attr('cx', 10)
+            .attr('cy', 10)
+            .attr('r', 9)
+            .attr('fill', '#47725f')
+            .attr('stroke', '#111')
+            .attr('stroke-width', 1.25);
+        svgEnter
+            .append('path')
+            .attr('d', 'M 7 19 L 10 28 L 13 19 Z')
+            .attr('fill', '#47725f')
+            .attr('stroke', '#111')
+            .attr('stroke-width', 1.25)
+            .attr('stroke-linejoin', 'round');
+
+        // inner logo, white, scaled to fit circle
+        const hScale = 0.28; // 40 * 0.28 = ~11.2px
+        const hg = svgEnter
+            .append('g')
+            .attr(
+                'transform',
+                `translate(${10 - 20 * hScale}, ${10 - 20 * hScale}) scale(${hScale})`,
+            )
+            .attr('fill', '#ffffff')
+            .attr('stroke', 'none');
+        hg.append('path').attr(
+            'd',
+            'm28.121 11.879-2.828 5.657-2.829-2.829zM11.879 28.121l2.828-5.657 2.829 2.829z',
+        );
+        hg.append('path').attr(
+            'd',
+            'M20 26a6 6 0 1 1 0-12 6 6 0 0 1 0 12Zm0-1.333a4.667 4.667 0 1 0 0-9.334 4.667 4.667 0 0 0 0 9.334Z',
+        );
+        hg.append('path').attr(
+            'd',
+            'M19.875 0C8.916 0 0 8.916 0 19.875c0 10.96 8.916 19.876 19.875 19.876 10.96 0 19.876-8.916 19.876-19.876C39.75 8.916 30.835 0 19.875 0Zm0 38.426c-10.228 0-18.55-8.322-18.55-18.55 0-10.23 8.322-18.551 18.55-18.551 10.229 0 18.55 8.322 18.55 18.55 0 10.229-8.321 18.55-18.55 18.55Z',
+        );
+        hg.append('path').attr(
```

Comment:
The Icon needs to be replaced. And should be shared somehow between Map and Sidebar header.